extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -520,7 +564,7 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		}
 	}
 
-	public async getInstalledPipPackages(pythonExePath?: string): Promise<PythonPkgDetails[]> {
+	public async getInstalledPipPackages(pythonExePath?: string, user: boolean = false): Promise<PythonPkgDetails[]> {
```

Comment:
Could we rework the "user" argument here? Maybe something like "checkUserPackages"? The current name is a little vague at a glance.